IDEAS/Buildings/Components/InterzonalAirFlow/Sim.mo

```diff
@@ -0,0 +1,60 @@
+within IDEAS.Buildings.Components.InterzonalAirFlow;
+model Sim
+  "Sim: Default model , either n50Tight or AirTight based on input in SimInfoManager"
+  parameter Real n50;
+
+  extends
+    IDEAS.Buildings.Components.InterzonalAirFlow.BaseClasses.PartialInterzonalAirFlown50(n50_int=if sim.interZonalAirFlowType<>IDEAS.BoundaryConditions.Types.InterZonalAirFlow.None then 0 else n50,
+    prescribesPressure=false, bou(azi=0));
+
+
+  Fluid.Interfaces.IdealSource airExfiltration(
+    redeclare package Medium = Medium,
+    control_m_flow=true,
+    allowFlowReversal=false,
+    control_dp=false) "Fixed air exfiltration rate" annotation (Placement(
+        transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=90,
+        origin={10,-50})));
+
+equation
+
+  connect(airExfiltration.m_flow_in, airInfiltration.m_flow_in)
+    annotation (Line(points={{2,-56},{-18,-56},{-18,-44}},  color={0,0,127}));
+  connect(airExfiltration.port_b, bou.ports[2])
+    annotation (Line(points={{10,-40},{10,0},{-1.77636e-15,0}},
+                                                      color={0,127,255}));
+  connect(airExfiltration.port_a, ports[2]) annotation (Line(points={{10,-60},{
+          10,-100},{2,-100},{2,-100}},
+                                    color={0,127,255}));
+  annotation (Documentation(revisions="<html>
+<ul>
+<li>
+March 17, 2020, Filip Jorissen:<br/>
```

Comment:
@kldjonge when copying code from others, best is to start a new revision history for that model

Reply:
Totally forgot about the revision history. I actually did not add any revision history to any of the models I adapted.